requirements_dev.txt

```diff
@@ -13,3 +13,5 @@ ecdsa==0.13
 base58==0.2.5
 bitcoin==1.1.42
 mpmath==1.0.0
+scrypt==0.8.0
```

Comment:
These 2 dependencies need to move into `requirements.txt` which is meant for deps for users of the project. `requirements_dev.txt` is meant for only neocore-dev-specific stuff.